packages/curve-ui-kit/src/shared/useApiStore.ts

```diff
@@ -0,0 +1,58 @@
+import { create, type StateCreator } from 'zustand'
+import { devtools } from 'zustand/middleware'
+import type curveApi from '@curvefi/api'
+import type lendingApi from '@curvefi/lending-api'
+import type { IChainId } from '@curvefi/lending-api/lib/interfaces'
+import type stablecoinApi from '@curvefi/stablecoin-api'
+
+export type CurveApi = typeof curveApi & { chainId: number }
+export type LendingApi = typeof lendingApi & { chainId: IChainId }
+export type StableApi = typeof stablecoinApi & { chainId: 1 }
+
+type State = {
+  curve: CurveApi | null
+  isLoadingCurve: boolean
+  lending: LendingApi | null
+  isLoadingLending: boolean
+  stable: StableApi | null
```

Comment:
We have been using crvUSD, loan and stablecoin as terms to refer to the crvUSD app - I've been trying to convert these references to `crvusd` as that's what is in the URL and the menus